### PR TITLE
new mobility variables

### PR DESCRIPTION
**Pull request by hanwzhang**
Updating wcc_setup to create new mobility variables

### CODE REVIEW

**Comment by hanwzhang** [comment]
I created kq45_cond_parq12, kq5_cond_parq12, and moves_two_quintiles (which now includes moving from par_q3 to kq5) both for chetty4y and for wcc.

File: notebooks/setup/wcc_setup.Rmd [hunk 1]
```diff
@@ -36,10 +43,43 @@ chetty4y <- chetty4y |>
...
+
+These are the mobility variables we want; we need to create the first three.
+
+- moves_two_quintiles, which is proportion of students that moved at least two quintiles in the income distribution.
+- kq45_cond_parq12
+- kq5_cond_parq12
+- kq5_cond_parq1
+
+```{r move 2 quintiles}
+chetty4y <- chetty4y |>
```
Comment (mjclawrence) [inline notebooks/setup/wcc_setup.Rmd:58]: Let's have two `moves_2_quintiles` variables: one that includes par_q3 and one that does not.

**Review by mjclawrence — CHANGES_REQUESTED** [review]
looks good. see comment on `moves_2_quintiles` variable.

**Comment by hanwzhang** [comment]
I created new variable moves_2_quintiles_from12

**Pull request merged**